packages/hash/tsup.config.ts

```
@@ -0,0 +1,24 @@
+import browserslistToEsbuild from 'browserslist-to-esbuild';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import-x/namespace)